generators/client/templates/angular/src/main/webapp/app/shared/language/_jhi-translate.directive.ts

```diff
@@ -7,7 +7,7 @@ import { Component, Input } from '@angular/core';
     selector: '[jhi-translate]',
     template: '<span [innerHTML]="jhiTranslate | translate:translateValues"></span>'
 })
-export class JhiTranslate {
+export class <%=jhiPrefixCapitalized%>Translate {
```

Comment:
So `JhiMissingTranslationHandler` and `JhiLanguageService` classes could also be changed from `jhiPrefix` to `Jhi` to achieve this.

I updated PR without doing this to only fix compile errors. They could be changed with another PR if needed.